analysed-packages/libnice/version-0.1.18/README.md

```diff
@@ -0,0 +1,18 @@
+## Download Location
+
+http://nice.freedesktop.org/releases/libnice-0.1.18.tar.gz
+
+## Package URL (purl)
+
+pkg:generic/libnice@0.1.18?download_url=http://nice.freedesktop.org/releases/libnice-0.1.18.tar.gz
+
+## Creator
+
+Sascha Dierberg
+
+## Reviewers
+
+The information was reviewed by:
+
+Mario Schuknecht
```

Comment:
In case of a dual licensed package we recently decided in OSSelot to provide in the README file the following phrase in a comment section:
## Comment
This package is dual licensed under MPL-1.1 or LGPL-2.1-only. To 
facilitate automatic use, a default license choice of LGPL-2.1-only is 
denoted in the Acknowledgment section of the OSS disclosure document. 
This shall not restrict the freedom of users to choose either 
MPL-1.1 or LGPL-2.1-only. Anyway, all license texts are provided.

This is was done recently and thus probably not known to you